routes/auths.js

```diff
@@ -14,4 +15,12 @@ router.post('/', validateLoginCredentials, async (req, res, next) => {
   }
 });
 
+router.post('/add', async (req, res) => {
```

Comment:
Please, can you wrap this code block in a `try {} catch() {}` and use `next` to handle error?